src/app-layer-htp.c

```diff
@@ -2519,6 +2519,9 @@ static void HTPConfigSetDefaultsPhase1(HTPCfgRec *cfg_prec)
     htp_config_register_response_complete(cfg_prec->cfg, HTPCallbackResponseComplete);
 
     htp_config_set_parse_request_cookies(cfg_prec->cfg, 0);
+#ifdef HAVE_HTP_CONFIG_SET_ALLOW_SPACE_URI
+    htp_config_set_allow_space_uri(cfg_prec->cfg, 1);
+#endif
```

Comment:
Curious: is this a change introduced by an update in libhtp? If so, wondering if we should indicate that in the docs...

Reply:
This API in libhtp was introduced long ago

Which docs are you talking about ?

Reply:
Suri docs. If it's old, then I don't think it's needed.